meta/README.md

```diff
@@ -0,0 +1,50 @@
+Mutant’s Mutation Operators
+===
+
+This directory contains all the [Mutation Operators](../docs/nomenclature.md#mutation-operator)[^mutation-operator] that Mutant can apply.
+
+The filenames mostly correspond to the names of code elements as used by the [parser](https://github.com/whitequark/parser) and [unparser](https://github.com/mbj/unparser) libraries. For the most part they are the same as keywords within Ruby, but here are some explanations of the less obvious ones:
+
+* `str` and `sym` are string and symbol literals that do not contain dynamic elements (`"I'm a dstring #{i_am_the_dynamic_element}"`)
+    * `dst` and `dsym` are the equivalents for strings and symbols which do
+* `regexp` is a regular expression without any options (`/like this/`)
+    * `regopt` is a regular expression with options (`/like this/i`)
+* a few come in regular and `-asgn` variants - the latter are the assignment versions. E.g., `||=` is the assignment version (`or_asgn.rb`) of `||`  (`or.rb`)
+* `gvar`, `cvar`, `ivar` and `lvar` refer to global (`$VERBOSE`), class (`@@debug`), instance (`@amount`) and local (`response`) variables, respectively.
+* `csend` is the conditional send operator (`&.`)
+* `cbase` is a top-level constant (`::Kernel` or `::Errno::ENOENT`), whereas `const` is a "normal" constant (`Kernel` or `Errno::ENOENT`)
+* `casgn` is an assignment to a constant (`VERSION = "0.1.0"`)
```

Comment:
This strikes me as very helpful to new contributors :)

Reply:
, or Things I Had To Learn When I Was Adding Metaclass Support 😁